### PR TITLE
Documentation x509v3_config.pod Authority Info Access, more examples,…

### PR DESCRIPTION
**Pull request by georg-x**
Hello,

Today I had to look up how to get Authority Info Access into an openssl CA. I was a dissapointend by the documentation, as there was no multi value example and no list of possible values.

Please tell me if anything is wrong with my suggested changes. This ist my first GIT/GITHUB/OpenSSL/OpenSource contribution at all, and maybe I got something wrong. Sorry, if so.

I consider the change to documentation as CLA: trivial. Which I also put into the commit meassage.

Kind Regards,
Georg

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man5/x509v3_config.pod
Comment (kaduk) [inline doc/man5/x509v3_config.pod:287]:
```suggestion
Possible values for access_id include B<OCSP> (OCSP responder),
```

The implementation and spec are both pretty generic about the accessMethod of the AccessDescription, so I'd be a little reluctant to imply that this list is exhaustive (which is what the "are" seems to do, for me).

File: doc/man5/x509v3_config.pod
Comment (kaduk) [inline doc/man5/x509v3_config.pod:297]: Arguably we should be using RFC 2606 reserved-for-documentation names, though I won't insist on making that change as part of this PR.

**Review by kaduk — COMMENTED** [review]
Other than the wording tweak suggested inline, this seems correct as far as I can tell.

File: doc/man5/x509v3_config.pod
Comment (georg-x) [reply doc/man5/x509v3_config.pod]: Updated x509v3_config.pod examples to example.com & example.org

**Review by kaduk — APPROVED** [review]
The new changes look good.
Note that we don't use merge commits in our repo, but IIRC that will just disappear when the committer rebases in preparation for pushing it to the official repo.
Thanks for taking on the optional task too!

**Comment by georg-x** [comment]
Thank you for approval and feedback. Is there anything left for me to do? Should I close the pull request? Or does that work out automatically?

**Comment by kaduk** [comment]
> Thank you for approval and feedback. Is there anything left for me to do? Should I close the pull request? Or does that work out automatically?

There is nothing left for you to do at the moment (basically, unless other review comments come in).  All commits need approval from an OTC member and a committer, then a 24-hour grace period to let anyone else comment before it can be merged.  I count for the "committer" approval, so we are waiting for an OTC review and the grace period, still.

**Comment by t8m** [comment]
Please note that this change is outside of what can be considered a CLA: trivial change. Please drop the CLA: trivial from the commit messages and submit an ICLA. https://www.openssl.org/policies/cla.html

**Comment by georg-x** [comment]
I sent the CLA as suggested.
I'm sorry, I messed up my local git repro and I got a headache with trying to fix the commit message to remove CLA:trivial.
You can still take my suggested change.... or maybe I'll have some time to try again with a fresh clone.

**Pull request closed without merging**
